### PR TITLE
fix bwc index tool for versions before 5.0

### PR DESCRIPTION
**Pull request by brwe**

### CODE REVIEW

**Comment by abeyad** [comment]
LGTM

**Comment by rjernst** [comment]
LGTM too.

File: dev-tools/create_bwc_index.py [hunk 1]
```diff
@@ -133,16 +133,19 @@ def start_node(version, release_dir, data_dir, repo_dir, tcp_port=DEFAULT_TRANSP
   logging.info('Starting node from %s on port %s/%s, data_dir %s' % (release_dir, tcp_port, http_port, data_dir))
   if cluster_name is None:
     cluster_name = 'bwc_index_' + version
-    
+  if version.startswith('5'):
```
Comment (clintongormley) [inline dev-tools/create_bwc_index.py]: perhaps this check should be for versions before 5 instead, otherwise we'll need to fix this for 6.0 as well

**Comment by brwe** [comment]
@clintongormley pushed another commit. Thanks all for the reviews.

File: dev-tools/create_bwc_index.py [hunk 2]
```diff
@@ -133,16 +133,19 @@ def start_node(version, release_dir, data_dir, repo_dir, tcp_port=DEFAULT_TRANSP
...
   cmd = [
     os.path.join(release_dir, 'bin/elasticsearch'),
-    '-Epath.data=%s' % data_dir,
-    '-Epath.logs=logs',
-    '-Ecluster.name=%s' % cluster_name,
-    '-Enetwork.host=localhost',
-    '-Etransport.tcp.port=%s' % tcp_port,
-    '-Ehttp.port=%s' % http_port,
-    '-Epath.repo=%s' % repo_dir
+    '%spath.data=%s' %(prefix, data_dir),
```
Comment (rjernst) [inline dev-tools/create_bwc_index.py]: nit: space after the second percent sign (on all these lines)

**Comment by rjernst** [comment]
LGTM

**Comment by brwe** [comment]
@rjernst pushed another commit, please check I was unsure where to put the spaces.

**Comment by clintongormley** [comment]
LGTM

**Pull request merged**